test/google_crawler_web/features/keywords/filter_keyword_test.exs

```diff
@@ -46,4 +48,23 @@ defmodule GoogleCrawlerWeb.FilterKeywordTest do
     |> assert_has(css(@selectors[:keyword_table_cell], text: keyword2.title))
     |> refute_has(css(@selectors[:keyword_table_cell], text: keyword3.title))
   end
+
+  feature "filters keywords which search results match amount and operation by the given params", %{session: session} do
+    user = insert(:user)
+    keyword1 = insert(:keyword, user: user)
+    keyword2 = insert(:keyword, user: user)
+
+    insert(:search_result, keyword: keyword1)
+    insert(:search_result, keyword: keyword1)
+
+    session
+    |> login_as(user)
+    |> visit("/keyword")
+    |> click(css(@selectors[:button_keyword_filter]))
+    |> click(css(@selectors[:option_result_count_filter]))
+    |> fill_in(css(@selectors[:input_result_count_filter]), with: "2")
+    |> click(css(@selectors[:button_submit_filter]))
+    |> assert_has(css(@selectors[:keyword_table_cell], text: keyword1.title))
+    |> refute_has(css(@selectors[:keyword_table_cell], text: keyword2.title))
```

Comment:
Same with the comment above, it hard to know why we don't see the `keyword2.title`, better to specify that in 

```
insert(:search_result, keyword: keyword1)
insert(:search_result, keyword: keyword2)
``

Reply:
same as I reply above, but for a clearer test, I will update the selector name to show that `equal` operator is selected